Whitelist Interfaces documentation (#8)
eProsima/Fast-RTPS-docs: docs/advanced.rst

This can end dropping samples.
To avoid this you can slow down the rate using :ref:`flow-controllers`.

Whitelist Interfaces
-------------------------------------

There could be situations where you want to block some network interfaces to avoid connections or sending data through them.

> To able this management exists the field *interface whitelist* in the transport descriptors, 

Reviewer: ```suggestion
This can be managed using the field *interface whitelist* in the transport descriptors, 
```

> and with them, you can set what interfaces do you want to filter to send or receive packets.

Reviewer: ```suggestion
and with them, you can set the interfaces you want to use to send or receive packets.
```

> The values that you must add to this list are the IPs of your machine in that networks. For example:

Reviewer: ```suggestion
The values on this list should match the IPs of your machine in that networks. For example:
```

> .. code-block:: c++

Reviewer: Please add xml configuration too
